sdk/core/azure-core/azure/core/tracing/abstract_span.py

```diff
@@ -73,6 +99,21 @@ def set_http_attributes(self, request, response=None):
         :type response: HttpResponse
         """
 
+    def get_trace_parent(self):
+        # type: () -> str
+        """Return traceparent string as defined in W3C trace context specification.
```

Comment:
Technically, this does not have to be a W3C trace context, does it?

Reply:
Yes, I would expect that implementation return a compatible W3C trace context here.

Reply:
What about tracing technologies that don't use the W3C format?

Reply:
Fixed the doc as discussed in live